app/components/TextEditor/index.ts

```diff
@@ -0,0 +1,5 @@
+import dynamic from 'next/dynamic';
+
+const TextEditor = dynamic(() => import('./TextEditor'), { ssr: false });
+
+export default TextEditor;
```

Comment:
We export TextEditor like this due to this warning:

<img width="1060" alt="image" src="https://github.com/user-attachments/assets/d6b82e65-22b7-4ae2-b943-54d0e7bef1dc">

However, this doesn't seemed to have gotten rid of it completely (it's logged less times though)

I want to see what this does in 'production' to understand if I need to do anything further to diagnose and fix this, or if it's a local development issue 🤔